### PR TITLE
version: bump claim file format version

### PR DESCRIPTION
**Pull request by jmontesi**
Depends-On: [test-network-function-claim#111](https://github.com/test-network-function/test-network-function-claim/pull/111)

### CODE REVIEW

File: version.json
Comment (edcdavid) [inline version.json:4]: we should also bump the parserTag to v0.4.0 .  Updating the claim format (https://github.com/test-network-function/parser/blob/562da1e644f1673dcaf545ad1225a5037108c555/html/index.js#L3) in the parser to 0.4.0 and create a new 0.4.0 release

Comment (jmontesi) [reply version.json]: Done. See [Parser#30](https://github.com/test-network-function/parser/pull/30). Will create the release once is merged.

**Review by sebrandon1 — APPROVED** [review]
Will this also be backported to `main`?

**Review by edcdavid — APPROVED** [review]

**Comment by jmontesi** [comment]
> Will this also be backported to `main`?

We will wait until `ginkgo_removal` is merged into `main`.

**Pull request merged**
